game.py

```diff
@@ -0,0 +1,54 @@
+from calculation.summation import Summation
+from calculation.calculation_class import Calculation
+
+
+class Game:
+    @staticmethod
+    def run_game():
+        summation = Summation(1, 10)
+
+        username = input("Здравей! Как се казваш?\n" "Моето име е: ")
+        print(f"Добре да се забавляваме, {username}. Нека да започнем с действие събиране!")
+        sum_not_correct_answer = 0
+        sum_correct_answer = 0
+
+        for i in range(1, 11):
+            print(f"Въпрос номер {i}:")
+            (question, answer) = summation.get_calculation()
+            print(f"На колко е равно:\n {question}")
+            user_answer = input("= ")
+            while True:
+                if user_answer == str(answer):
+                    print(f"Поздравления, твоят отговор {answer} е правилен.")
+                    sum_correct_answer += 1
+                    break
+                elif user_answer != str(answer):
```

Comment:
Here we can simplify it. Because `user_answer != str(answer)` is the opposite of `user_answer == str(answer)` it is sufficient to just write `else:`